### PR TITLE
Create win_susp_vuln_cve_2022_21919_or_cve_2021_34484.yml

### PR DESCRIPTION
**Pull request by sorchaa**
During exploitation of this vuln, It appears when the directory \Users\TEMP is created, two logs (providername:Microsoft-Windows-User Profiles Service) with eventid 1511 and 1515 are created. Viewed on 2008 Server

### CODE REVIEW

**Comment by frack113** [comment]
HI from [Microsoft-Windows-User%20Profiles%20Service.xml](https://github.com/nasbench/EVTX-ETW-Resources/blob/main/ETWProvidersManifests/WindowsServer/2022/WindowsServer2022_21H2_Standard_20220719_20348.825/WEPExplorer/Microsoft-Windows-User%20Profiles%20Service.xml) : 
[eventid 1511](https://github.com/nasbench/EVTX-ETW-Resources/blob/45fd5be71a51aa518b1b36d4e1f36af498084e27/ETWProvidersManifests/WindowsServer/2022/WindowsServer2022_21H2_Standard_20220719_20348.825/WEPExplorer/Microsoft-Windows-User%20Profiles%20Service.xml#L742) make sense
[eventid 1515](https://github.com/nasbench/EVTX-ETW-Resources/blob/45fd5be71a51aa518b1b36d4e1f36af498084e27/ETWProvidersManifests/WindowsServer/2022/WindowsServer2022_21H2_Standard_20220719_20348.825/WEPExplorer/Microsoft-Windows-User%20Profiles%20Service.xml#L794) could give FP

File: rules/windows/builtin/system/win_susp_vuln_cve_2022_21919_or_cve_2021_34484.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+title: Suspicious usage of CVE 2022_21919 or CVE_2021_34484
+id: 52a85084-6989-40c3-8f32-091e12e13f09
+status: test
```
[the remark this answers is not in the document]
Comment (sorchaa) [reply rules/windows/builtin/system/win_susp_vuln_cve_2022_21919_or_cve_2021_34484.yml]: i replaced by experimental

File: rules/windows/builtin/system/win_susp_vuln_cve_2022_21919_or_cve_2021_34484.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+description: During exploitation of this vuln, It appears when the directory \Users\TEMP is created, two logs (providername:Microsoft-Windows-User Profiles Service) with eventid 1511 and 1515 are created. Viewed on 2008 Server
+author: Sorcha
+references:
+  - https://packetstormsecurity.com/files/166692/Windows-User-Profile-Service-Privlege-Escalation.html
+date: 22/08/16
+logsource:
+  product: windows
+  service: Profile Service
```
[the remark this answers is not in the document]
Comment (sorchaa) [reply rules/windows/builtin/system/win_susp_vuln_cve_2022_21919_or_cve_2021_34484.yml]: I replaced by service: application

File: rules/windows/builtin/system/win_susp_vuln_cve_2022_21919_or_cve_2021_34484.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+detection:    
+      EventID:
+          - 1511   
+          - 1515                      
+      System.ProviderName: 'Microsoft-Windows-User Profiles Service'
```
[the remark this answers is not in the document]
Comment (sorchaa) [reply rules/windows/builtin/system/win_susp_vuln_cve_2022_21919_or_cve_2021_34484.yml]: I replaced by Providername

**Comment by sorchaa** [comment]
Hi,
I did the modifications you suggested.

File: rules/windows/builtin/system/win_susp_vuln_cve_2022_21919_or_cve_2021_34484.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+status: Experimental
+description: During exploitation of this vuln, It appears when the directory \Users\TEMP is created, two logs (providername:Microsoft-Windows-User Profiles Service) with eventid 1511 and 1515 are created. Viewed on 2008 Server
+author: Sorcha
+references:
+  - https://packetstormsecurity.com/files/166692/Windows-User-Profile-Service-Privlege-Escalation.html
+date: 22/08/16
+logsource:
+  product: windows
+  service: application
+detection:
```
Comment (frack113) [inline rules/windows/builtin/system/win_susp_vuln_cve_2022_21919_or_cve_2021_34484.yml:12]:
```yaml
detection: 
    selection:
        EventID: 1511                      
        ProviderName: 'Microsoft-Windows-User Profiles Service'
    condition: selection
```

**Review by frack113 — CHANGES_REQUESTED** [review]

File: rules/windows/builtin/system/win_susp_vuln_cve_2022_21919_or_cve_2021_34484.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+title: Suspicious usage of CVE 2022_21919 or CVE_2021_34484
```
Comment (frack113) [inline rules/windows/builtin/system/win_susp_vuln_cve_2022_21919_or_cve_2021_34484.yml:1]: `title: Suspicious Usage of CVE 2022_21919 or CVE_2021_34484`
https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide#title

File: rules/windows/builtin/system/win_susp_vuln_cve_2022_21919_or_cve_2021_34484.yml [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+id: 52a85084-6989-40c3-8f32-091e12e13f09
+status: Experimental
+description: During exploitation of this vuln, It appears when the directory \Users\TEMP is created, two logs (providername:Microsoft-Windows-User Profiles Service) with eventid 1511 and 1515 are created. Viewed on 2008 Server
+author: Sorcha
+references:
+  - https://packetstormsecurity.com/files/166692/Windows-User-Profile-Service-Privlege-Escalation.html
+date: 22/08/16
```
Comment (frack113) [inline rules/windows/builtin/system/win_susp_vuln_cve_2022_21919_or_cve_2021_34484.yml:8]: `date: 2022/08/16`
https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide#date

**Review by frack113 — CHANGES_REQUESTED** [review]

**Comment by frack113** [comment]
You can run `python3 tests/test_rules.py` to test before push

**Comment by sorchaa** [comment]
Hi,
It's updated.

**Comment by sorchaa** [comment]
test_rules.py passed

**Pull request merged**